packages/worker/src/api/routes/global/users.ts

```diff
@@ -72,7 +72,8 @@ router
   )
 
   .get("/api/global/users", auth.builderOrAdmin, controller.fetch)
-  .post("/api/global/users/search", auth.builderOrAdmin, controller.search)
+  // search can be used by any user now, to retrieve users for user column
+  .post("/api/global/users/search", controller.search)
```

Comment:
Is the authorised middleware added by default?

Reply:
For the worker - all endpoints require authentication by default - thats why all of the global stuff is in here, while the server doesn't strictly require permissions for everything.

I added a test case to show this as well, gets 403 if not logged in on the search endpoint.